examples/ml-pipelines-node-classification.ipynb

```diff
@@ -0,0 +1,1415 @@
+   "metadata": {
+    "colab": {
+     "background_save": true
+    },
+    "id": "Kvr9iU4HdoTv"
+   },
+   "outputs": [],
+   "source": [
+    "# Perform the actual training\n",
+    "model_fastrp, stats_fastrp = node_pipeline_fastrp.train(\n",
+    "    G,\n",
+    "    targetNodeLabels=[\"Paper\"],\n",
+    "    modelName=\"cora-pipeline-model-fastrp\",\n",
+    "    targetProperty=\"subject\",\n",
+    "    metrics=[\"F1_WEIGHTED\"],\n",
+    "    randomSeed=42,\n",
+    "    concurrency=4,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `F1_WEIGHTED` metrics is better with embeddings:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {
+    "colab": {
+     "background_save": true
+    },
+    "id": "_Q3zMmpCd2o0"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.8323028609950918\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(stats_fastrp[\"modelInfo\"][\"metrics\"][\"F1_WEIGHTED\"][\"test\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The classification using `predict_stream` can be run in the same way:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "background_save": true
+    },
+    "id": "vgfN-ppHw4Rp"
+   },
+   "outputs": [],
+   "source": [
+    "predicted_fastrp = model_fastrp.predict_stream(\n",
+    "    G,\n",
+    "    modelName=\"cora-pipeline-model-fastrp\",\n",
+    "    includePredictedProbabilities=True,\n",
+    "    targetNodeLabels=[\"UnclassifiedPaper\"],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "10\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(len(predicted_fastrp))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "PNbRL2y84F6Q"
+   },
+   "source": [
+    "Instead of streaming the results, the prediction can be run in `mutate` mode to be more performant, especially when the predicted values are used multiple times. The predicted nodes can be retrieved using the `streamNodeProperty` method with the `UnclassifiedPaper` class."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "background_save": true
+    },
+    "id": "_HVufWHbxIoE"
+   },
+   "outputs": [],
+   "source": [
+    "model_fastrp.predict_mutate(\n",
+    "    G,\n",
+    "    mutateProperty=\"predictedClass\",\n",
+    "    modelName=\"cora-pipeline-model-fastrp\",\n",
+    "    predictedProbabilityProperty=\"predictedProbabilities\",\n",
+    "    targetNodeLabels=[\"UnclassifiedPaper\"],\n",
+    ")\n",
+    "\n",
+    "predicted_fastrp = gds.graph.streamNodeProperty(G, \"predictedClass\", [\"UnclassifiedPaper\"])"
```

Comment:
Let's use the modernised procedures for streaming node properties: `gds.graph.nodeProperty.stream`

```suggestion
    "Instead of streaming the results, the prediction can be run in `mutate` mode to be more performant, especially when the predicted values are used multiple times. The predicted nodes can be retrieved using the `gds.graph.nodeProperty.stream` procedure with the `UnclassifiedPaper` class."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "colab": {
     "background_save": true
    },
    "id": "_HVufWHbxIoE"
   },
   "outputs": [],
   "source": [
    "model_fastrp.predict_mutate(\n",
    "    G,\n",
    "    mutateProperty=\"predictedClass\",\n",
    "    modelName=\"cora-pipeline-model-fastrp\",\n",
    "    predictedProbabilityProperty=\"predictedProbabilities\",\n",
    "    targetNodeLabels=[\"UnclassifiedPaper\"],\n",
    ")\n",
    "\n",
    "predicted_fastrp = gds.graph.nodeProperty.stream(G, \"predictedClass\", [\"UnclassifiedPaper\"])"
```